packages/main/src/Dialog.ts

```diff
@@ -85,6 +85,8 @@ const ICON_PER_STATE: Record<ValueStateWithIcon, string> = {
  * <li>content - Used to style the content of the component</li>
  * <li>footer - Used to style the footer of the component</li>
  * </ul>
+ * <b>Note:</b> When ui5-bar is used in the Header or in the Footer additional styling is needed. The default Dialog's paddings should be removed.
```

Comment:
I suggest to simplify:
<b>Note:</b> When a <code>ui5-bar</code> is used in the header or in the footer, you should remove the default dialog's paddings.
 * For more information see the sample "Bar in Header/Footer".